build.sbt

```diff
@@ -1,22 +1,16 @@
-scalaVersion := "2.11.8"
+scalaVersion := "2.12.4"
 
 name := "kamon-spm-example"
 
-organization := "com.sematext"
-
 version := "0.1"
 
-aspectjSettings
-
-javaOptions in run <++= AspectjKeys.weaverOptions in Aspectj
-
 fork in run := true
 
 libraryDependencies ++= Seq(
-  "io.kamon" %% "kamon-core" % "0.6.7",
-  "io.kamon" %% "kamon-spm" % "0.6.7",
-  "io.kamon" %% "kamon-akka-2.5" % "0.6.7",
-  "io.kamon" %% "kamon-system-metrics" % "0.6.7",
+  "io.kamon" %% "kamon-core" % "1.1.0",
+  "io.kamon" %% "kamon-spm" % "1.1.2",
+  "io.kamon" %% "kamon-akka-2.5" % "1.1.0",
+  "io.kamon" %% "kamon-system-metrics" % "1.0.0",
   "io.kamon" %% "kamon-log-reporter" % "0.6.7",
```

Comment:
This one really remained at 0.6.7, not 1.something?

Reply:
No. 0.6.8 is the latest. And I am not sure if this is compatible with 1.x. Since we use SPM Reporter in this example, I will remove this dependency. I see LogReporter.scala uses old API and I think it is not loaded.